Amend ZIP1014 to provide a Discretionary Budget for MGRC (#668)
zcash/zips: zip-1014.rst

public good of the Zcash ecosystem, to the extent that such teams are available
and effective.

The funds SHALL be received and administered by ZF. ZF MUST disburse them for
"Major Grants" and expenses reasonably related to the administration of Major 

> Grants, but subject to the following additional constraints:

Author: As originally drafted, ZIP 1014 restricts ZF from disbursing MG Slice funds for anything other than Major Grants. To be able to disburse funds from the Discretionary Budget on behalf of MGRC (aka ZCG), we need to relax this restriction.

1. These funds MUST only be used to issue Major Grants to external parties
   that are independent of ZF, and to pay for expenses reasonably related to 
   the administration of Major Grants. They MUST NOT be used by ZF for its 
   internal operations and direct expenses not related to administration of 

>    Major Grants. Additionally, BP, ECC, and ZF are ineligible to receive 

Author: As with the added text on lines 213-214, this change is required in order to allow ZF to disburse funds from the Discretionary Budget.

   an immediate family relationship with any of the above. The ZF SHALL continue
   to operate the Community Advisory Panel and SHOULD work toward making it more
   representative and independent (more on that below).
   
8. From 1st January 2022, a portion of the MG Slice shall be allocated to a 
   Discretionary Budget, which may be disbursed for expenses reasonably related 
   to the administration of Major Grants. The amount of funds allocated to the 
   Discretionary Budget SHALL be decided by the ZF's Community Advisory Panel or 
   successor process. Any disbursement of funds from the Discretionary Budget 

>    MUST be approved by the Major Grant Review Committee. Expenses related to the 

Author: The stricture that "Any disbursement of funds from the Discretionary Budget MUST be approved by the Major Grant Review Committee." ensures that the authority to disburse any funds from the Discretionary Budget lies with MGRC.

>    administration of Major Grants include, without limitation the following:

Author: The use of "without limitation" here indicates that the list below is not intended to be exhaustive or restrictive, merely a list of _examples_.

> 8. From 1st January 2022, a portion of the MG Slice shall be allocated to a 
>    Discretionary Budget, which may be disbursed for expenses reasonably related 
>    to the administration of Major Grants. The amount of funds allocated to the 
>    Discretionary Budget SHALL be decided by the ZF's Community Advisory Panel or 
>    successor process. Any disbursement of funds from the Discretionary Budget 
>    MUST be approved by the Major Grant Review Committee. Expenses related to the 
>    administration of Major Grants include, without limitation the following:

Reviewer: ```suggestion
8. From 1st January 2022, a portion of the MG Slice shall be allocated to a 
   Discretionary Budget, which may be disbursed for expenses reasonably related 
   to the administration of Major Grants. Constraints on the amount of funds 
   allocated to the Discretionary Budget SHALL be decided by the ZF's Community 
   Advisory Panel or successor process. Any disbursement of funds from the 
   Discretionary Budget MUST be approved by the Major Grant Review Committee. 
   Expenses related to the administration of Major Grants include, without 
   limitation the following:
```

Author: This language was drafted by legal counsel, so I'm going to respectfully reject the suggested change.

>    * Paying third party vendor for services related to domain name registration,  or

Reviewer 2: ```suggestion
   * Paying third party vendors for services related to domain name registration, or
```

     website design, website hosting and administration for the Major Grant 

>      Review Committee.

Reviewer: >website hosting and administration 

Is this `administration of websites` for MGRC?

Author: Yes. See https://zcashcommunitygrants.org/

>    * Paying third party vendor for services related to domain name registration, 
>      website design, website hosting and administration for the Major Grant 
>      Review Committee.

Reviewer: ```suggestion
   * Paying third party vendor for services related to domain name registration or 
     to design, hosting and administration of websites for the Major Grant 
     Review Committee.
```

Author: As the text says, these are merely examples so I have no problem with this suggestion.

Author: ```suggestion
   * Paying third party vendor for services related to domain name registration,  or
     the design, website hosting and administration of websites for the Major Grant 
     Review Committee.
```
As this is a list of (non-restrictive) examples, I see no issue with wordsmithing this sentence.

>    * Paying independent consultant to develop requests for proposals that align 

Reviewer 3: Optional: grammar?

```suggestion
   * Paying independent consultants to develop requests for proposals that align 
```

     with the Major Grants program.
   * Paying independent consultant for expert review of grant applications.
   * Paying for sales and marketing services to promote the Major Grants 
     program.
   * Paying third party consultant to undertake activities that support the 

>      purpose of the Major Grants program. 

Reviewer: Can we make this more concrete?

Author: Well, this is a list of examples, and is therefore _not_ intended to be exhaustive or restrictive. Making it more concrete would implicitly be restrictive.

Reviewer: ok

>    * Paying independent consultant to develop requests for proposals that align 
>      with the Major Grants program.
>    * Paying independent consultant for expert review of grant applications.
>    * Paying for sales and marketing services to promote the Major Grants 
>      program.
>    * Paying third party consultant to undertake activities that support the 
>      purpose of the Major Grants program. 

Reviewer 3: Optional: grammar?

```suggestion
   * Paying independent consultants to develop requests for proposals that align 
     with the Major Grants program.
   * Paying independent consultants for expert review of grant applications.
   * Paying for sales and marketing services to promote the Major Grants 
     program.
   * Paying third party consultants to undertake activities that support the 
     purpose of the Major Grants program. 
```
